detection-rules/link_undisclosed_recipients_credphish.yml

```diff
@@ -20,7 +20,14 @@ source: |
           )
       )
   )
-tags: 
-  - "Suspicious headers"
-  - "Suspicious link"
+attack_types:
+  - "Credential Phishing"
+tactics_and_techniques:
+  - "Evasion"
```

Comment:
```suggestion
```

Reply:
I personally view the technique of masking recipients as evasion
      // No Recipients
      length(recipients.to) == 0
      or all(recipients.to, .display_name == "Undisclosed recipients")